client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -741,18 +750,21 @@ const mapStateToProps = ( state, ownProps ) => {
 	const { site } = ownProps;
 	// SEO Tools are available with Business plan on WordPress.com, and with Premium plan on Jetpack sites
 	const isAdvancedSeoEligible = site && site.plan && hasBusinessPlan( site.plan );
-	const siteId = get( site, 'ID', 0 );
+	const siteId = getSelectedSiteId( state );
 	const jetpackManagementUrl = getSiteOption( state, siteId, 'admin_url' );
 	const jetpackVersionSupportsSeo = isJetpackMinimumVersion( state, siteId, '4.4-beta1' );
 	const isAdvancedSeoSupported = site && ( ! site.jetpack || ( site.jetpack && jetpackVersionSupportsSeo ) );
 
 	return {
+		siteId: siteId,
```

Comment:
You can use the object literal property value shorthand here, which allows you to change this line to:

```
siteId,
```